### ISSUE TITLE
Migration

### ISSUE THREAD

**Issue by HansPeterDurand** [home-assistant/home-assistant.github.io#36257]
### Feedback

Good afternoon,
Today I migrated from the HACS based "The Things Network" Integration to the news Home Assistant based Ingetration. For me, used the follwoing procedure/plan:

Procedure/plan

	1. backup the Home Assistant installation (possibly copy to PC via samba “Add on”)
	2. deactivate the automation (in my case “Post - Mailbox”) slide switch to the left
	3. deactivate the user-defined/HACS based integration “The Things Network”
	4. delete the user-defined/HACS based integration “The Things Network”.
	5. delete/remove the integration in HACS
	6. restart the Home Assistant
	7. the further steps for configuring the new integration are described under this link: 
	    https://www.home-assistant.io/integrations/thethingsnetwork
		a. Generate a new API key on the TTN console “console.cloud.thethings.network”: When generating the new API key,
		   define the “expiration time”: e.g.: 31.12.2029). As soon as the key has been generated, copy it away and confirm 
		   that it has been copied.
		b. When adding the integration, the following information is requested:
		
		Host: eu1.cloud.thethings.network
		Application ID: ms24-letterbox 
		Application Key: see above
		
	9. after successful registration: the device and its entities can be checked. Device an entity are the same as before!
	10. perform final test.

In conclusion: Life is boring everything works 😊

Regards
Hans-Peter Durand

### URL

https://www.home-assistant.io/integrations/thethingsnetwork/

### Version

2024.12.1

### Additional information

Living in Europe/Switzerland
Gateway: LPS8v2 -- LoRaWAN Indoor Gateway from Dragino
Sensor: SN50v3-LB/LS -- LoRaWAN Sensor Node User Manual - DRAGINO

**Comment by frenck** [comment]
Hi there @HansPeterDurand 👋 

Thanks for the procedure, but I'm not entirely sure what to do with it? The HACS integration isn't something that is supported from our end.

For that reason, I'll go ahead and go close this feedback; nevertheless, I'm happy to learn that you have found a way to get it working for your usecase.

../Frenck

**Comment by angelnu** [comment]
It might help others migrating. I will unarchive the hacs integration, add a reference there and archive again.

**Comment by HansPeterDurand** [comment]
Good morning everybody,
Thank you for your comment. Well, when i wrote program code, i used languaes like Fortran, Assembler, Bliss etc. Due to that fact i deeply estimate your work. I thought my "procedure" is al little contribution to your great work.

Best regards

Hans Peter (64 years old)
